packages/hub/services/scheduled-payments/fetcher.ts

```diff
@@ -84,9 +100,16 @@ export default class ScheduledPaymentsFetcherService {
         ORDER BY last_failed_payment_attempt.started_at ASC NULLS FIRST
         LIMIT ${limit};`;
 
-    // Retrieve scheduledPayment prisma object based on filtered id
     return prisma.scheduledPayment.findMany({ where: { id: { in: results.map((result) => result.id) } } });
   }
+
+  calculateRetryBackoffsInMinutes() {
+    let fixedPart = [0, 5, 60, 360]; // Retry immediately, then after 5 minutes, 1 hour, 6 hours
+    let fixedPartSum = fixedPart.reduce((a, b) => a + b, 0);
+    let variablePartChunksCount = Math.floor((this.validForDays * 24 * 60 - fixedPartSum) / 720);
```

Comment:
supernit: but it would be nice to add meaning to some of the values, like:
 ```ts
const  oneDayInMinutes = 24 * 60`;
const  (halfDayInMinutes or twelveHoursInMin) = 720; and then reuse this value 
...
```
or you could have one single object with all of this values
```ts
const time = {
    fiveMin: 5,
    oneHour: 60,
    sixHours: 360,
    oneDay: 24 * 60,
    twelveHours: 720
}
```